AnalyticsServicePreferencesStorageQueueInboundProcessorAdapter/index.ts

```diff
@@ -0,0 +1,64 @@
+import { Context } from "@azure/functions";
+import { RetrievedServicePreference } from "@pagopa/io-functions-commons/dist/src/models/service_preference";
+import { NonEmptyString } from "@pagopa/ts-commons/lib/strings";
+import * as t from "io-ts";
+import * as KP from "../utils/kafka/KafkaProducerCompact";
+import { ValidableKafkaProducerConfig } from "../utils/kafka/KafkaTypes";
+import { getConfigOrThrow, withTopic } from "../utils/config";
+import * as KA from "../outbound/adapter/kafka-outbound-publisher";
+import * as EA from "../outbound/adapter/empty-outbound-publisher";
+import * as TA from "../outbound/adapter/tracker-outbound-publisher";
+import * as PDVA from "../outbound/adapter/pdv-id-outbound-enricher";
+import { OutboundPublisher } from "../outbound/port/outbound-publisher";
+import { getAnalyticsProcessorForDocuments } from "../businesslogic/analytics-publish-documents";
+import { OutboundEnricher } from "../outbound/port/outbound-enricher";
+import { servicePreferencesAvroFormatter } from "../utils/formatter/servicePreferencesAvroFormatter";
+
+export type RetrievedServicePreferenceWithMaybePdvId = t.TypeOf<
+  typeof RetrievedServicePreferenceWithMaybePdvId
+>;
+const RetrievedServicePreferenceWithMaybePdvId = t.intersection([
+  RetrievedServicePreference,
+  t.partial({ userPDVId: NonEmptyString })
+]);
+
+const config = getConfigOrThrow();
+
+const servicePreferencesConfig = withTopic(
+  config.servicePreferencesKafkaTopicConfig.SERVICE_PREFERENCES_TOPIC_NAME,
+  config.servicePreferencesKafkaTopicConfig
+    .SERVICE_PREFERENCES_TOPIC_CONNECTION_STRING
+)(config.targetKafka);
+
+const servicePreferencesTopic = {
+  ...config.targetKafka,
+  messageFormatter: servicePreferencesAvroFormatter()
+};
```

Comment:
```suggestion
const servicePreferencesTopic = {
  ...servicePreferencesConfig,
  messageFormatter: servicePreferencesAvroFormatter()
};
```

Reply:
Fixed in [19d305e](https://github.com/pagopa/io-functions-elt/pull/85/commits/19d305e58b9073b3625d858bd3ce34a608fdd700)